nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-restapi-processors/src/test/groovy/org/apache/nifi/processors/elasticsearch/PaginatedJsonQueryElasticsearchTest.groovy

```diff
@@ -61,6 +61,21 @@ class PaginatedJsonQueryElasticsearchTest extends AbstractPaginatedJsonQueryElas
         reset(runner)
 
 
+        // NIFI-11430 test that 1 flow file is output when hits output per response with output empty flow file set to true
+        runner.setProperty(AbstractJsonQueryElasticsearch.OUTPUT_NO_HITS, "true")
```

Comment:
Once a property has been set for the processor on the `runner`, it doesn't get reset/changed unless explicitly modified by the test code - the existing tests were presumably using the default of not outputting `no_hits`, the new tests are setting this to `true` and leaving it that way with little/no other assertion changes (i.e. we basically run the exact same test and aseertions twice)

Be sure to set `runner.setProperty(AbstractJsonQueryElasticsearch.OUTPUT_NO_HITS, "false")` for each of the existing sets of tests here before the `runOnce(runner)` calls

Even better, consider refactoring to loop through the different values (`OUTPUT_NO_HITS`, but possibly `SEARCH_RESULTS_SPLIT` too), similar to how you've done it in `AbstractPaginatedJsonQueryElasticsearchTest.groovy`

Reply:
👍🏻